www-client/netsurf/netsurf-3.8.recipe

```diff
@@ -1,19 +1,18 @@
-SUMMARY="A multi-platform web browser"
-DESCRIPTION="Small as a mouse, fast as a cheetah and available for free. NetSurf is a \
-multi-platform web browser for RISC OS, UNIX-like platforms (including Linux), \
-Mac OS X, and more.\
+SUMMARY="lightweight multi-platform web browser"
```

Comment:
Suggestion:
```diff
-SUMMARY="lightweight multi-platform web browser"
+SUMMARY="A lightweight multi-platform web browser"
```
That's because haikuporter complains (in strict mode):
```
Error: SUMMARY must start with a capital letter (/nfs/haikuports/www-client/netsurf/netsurf-3.8.recipe).
```

Reply:
Fixed.